### PR TITLE
HC palette update + exceptions for emphasis

### PR DESCRIPTION
**Pull request by Juliusschaeper**
- Added updated HC palette

Added exceptions: 
- onEmphasis to be dark
- adjusted border style for btnPrimary.

### CODE REVIEW

File: data/colors_v2/themes/dark_high_contrast.ts [hunk 1]
```diff
@@ -4,59 +4,118 @@ import dark from './dark'
...
+    onEmphasis:get('scale.gray.9')
   },
   border: {
     default: get('scale.gray.5'),
     muted: get('scale.gray.5')
   },
   neutral: {
     emphasis: get('scale.gray.6')
+  },
+  btn.primary: {
```
Comment (Juliusschaeper) [inline data/colors_v2/themes/dark_high_contrast.ts:116]: Not sure if this is the correct way to call btn primary…?

File: data/colors_v2/themes/dark_high_contrast.ts [hunk 2]
```diff
@@ -4,59 +4,118 @@ import dark from './dark'
...
+    '#d23d91', 
+    '#b72c7d', 
+    '#9c1d6a']
 }
 
 const exceptions = {
   fg: {
-    muted: get('scale.gray.1')
+    muted: get('scale.gray.1'),
+    onEmphasis:get('scale.gray.9')
```
Comment (colebemis) [inline data/colors_v2/themes/dark_high_contrast.ts:107]:
```suggestion
    onEmphasis: get('scale.gray.9')
```

File: data/colors_v2/themes/dark_high_contrast.ts [hunk 3]
```diff
@@ -4,59 +4,118 @@ import dark from './dark'
...
   },
   border: {
     default: get('scale.gray.5'),
     muted: get('scale.gray.5')
   },
   neutral: {
     emphasis: get('scale.gray.6')
+  },
+  btn.primary: {
+    border: get('scale.green.1'),
```
Comment (colebemis) [inline data/colors_v2/themes/dark_high_contrast.ts:117]:
```suggestion
  btn: {
    primary: {
      border: get('scale.green.1')
    }
```

File: data/colors_v2/themes/dark_high_contrast.ts (hunk 1 above)
Comment (colebemis) [reply data/colors_v2/themes/dark_high_contrast.ts:116]: This should fix it: https://github.com/primer/primitives/pull/137#discussion_r650254548

File: data/colors_v2/themes/dark_high_contrast.ts [hunk 4]
```diff
@@ -4,59 +4,120 @@ import dark from './dark'
...
   },
   neutral: {
     emphasis: get('scale.gray.6')
+  },
+  btn: {
+    primary: {
+      border: get('scale.green.1')
+    }
+    hoverBorder: get('scale.green.1')
   }
```
Comment (colebemis) [inline data/colors_v2/themes/dark_high_contrast.ts:121]:
```suggestion
  },
  btn: {
    primary: {
      border: get('scale.green.1'),
      hoverBorder: get('scale.green.1')
    }
  }
```

**Review by colebemis — APPROVED** [review]

**Review by simurai — APPROVED** [review]

**Comment by Juliusschaeper** [comment]
Thanks for the help @colebemis @simurai 🎉

**Pull request merged**